docs/src/content/en/docs/workflows/using-with-agents-and-tools.mdx

```diff
@@ -5,346 +5,119 @@ description: "Steps in Mastra workflows provide a structured way to manage opera
 
 # Agents and Tools
```

Comment:
In the Agent section, you have a page called Using Tools, so maybe this should be using Tools and Agents or using Tools & Agents (if using &, we should be consistent about that in all headings)